### PR TITLE
ruRu update

### PR DESCRIPTION
**Pull request by andaril**

### CODE REVIEW

File: Nightmare/Locales/ruRU.lua [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
 L = BigWigs:NewBossLocale("Il'gynoth", "ruRU")
 if L then
-	--L.custom_off_deathglare_marker = "Deathglare Tentacle marker"
-	--L.custom_off_deathglare_marker_desc = "Mark Deathglare Tentacles with {rt6}{rt5}{rt4}{rt3}, requires promoted or leader.\n|cFFFF0000Only 1 person in the raid should have this enabled to prevent marking conflicts.|r\n|cFFADFF2FTIP: If the raid has chosen you to turn this on, having nameplates enabled or quickly mousing over the spears is the fastest way to mark them.|r"
+	L.custom_off_deathglare_marker = "Маркировка Щупалец смертоносного взгляда"
+	L.custom_off_deathglare_marker_desc = "Помечать Щупальца смертоносного взгляда метками {rt6}{rt5}{rt4}{rt3}, требуется быть помощником или лидером рейда.\n|cFFFF0000Только один человек в рейде должен включить данную опцию чтобы избежать конфликтов.|r\n|cFFADFF2FПодсказка: Проведите мышкой над щупальцами, чтобы быстрее отметить их.|r"
 
-	--L.bloods_remaining = "%d |4Blood:Bloods; remaining"
+	L.bloods_remaining = "%d |4Кровь:Крови; осталось" --huh?
```
Comment (funkydude) [inline Nightmare/Locales/ruRU.lua]: Short name for http://ru.wowhead.com/npc=105721/nightmare-ichor

Comment (andaril) [inline Nightmare/Locales/ruRU.lua]: I understood for that npc is this. :) 
But I haven't found any info in wiki for pluralization of locale strings. So I just silly translated this. I thinks it's okay?

Comment (funkydude) [inline Nightmare/Locales/ruRU.lua]: You don't have to make it singular/plural, that's completely optional. You can replace it with just one word if you wish.

Comment (andaril) [inline Nightmare/Locales/ruRU.lua]: But how it works?
If I want for singular form `Кровь` and `Крови` for plural. And `осталась` `осталось` for same.
```
"%d |1Кровь:Крови; |1осталась:осталось;"
```
?

Comment (funkydude) [inline Nightmare/Locales/ruRU.lua]: I don't know what `|1` does. Wouldn't this work?
`"%d |4Кровь осталась:Крови осталось;"`

Comment (Wetxius) [inline Nightmare/Locales/ruRU.lua]: Maybe just use 
L.bloods_remaining = "Осталось крови: %d"

Comment (Wetxius) [inline Nightmare/Locales/ruRU.lua]: Maybe just use 
L.bloods_remaining = "Осталось крови: %d"

File: Nightmare/Locales/ruRU.lua [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+	L.forces = "Силы кошмара"
+	L.bramblesSay = "Колючки рядом с %s"
 end
 
 L = BigWigs:NewBossLocale("Elerethe Renferal", "ruRU")
 if L then
-	--L.isLinkedWith = "%s is linked with %s"
-	--L.yourLink = "You are linked with %s"
-	--L.yourLinkShort = "Linked with %s"
+	L.isLinkedWith = "%s связан(а) с %s"
```
Comment (funkydude) [inline Nightmare/Locales/ruRU.lua]: Is `связан(а)` intended? Seems weird.

Comment (andaril) [inline Nightmare/Locales/ruRU.lua]: He is `связан`
She is `связана`

I can leave only male variant

**Pull request merged**